sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/ServiceStore.java

```diff
@@ -0,0 +1,187 @@
+package com.mesosphere.sdk.scheduler.multi;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.mesosphere.sdk.scheduler.AbstractScheduler;
+import com.mesosphere.sdk.scheduler.SchedulerUtils;
+import com.mesosphere.sdk.storage.Persister;
+import com.mesosphere.sdk.storage.PersisterException;
+import com.mesosphere.sdk.storage.PersisterUtils;
+import com.mesosphere.sdk.storage.StorageError.Reason;
+
+/**
+ * Default implementation of persistent storage which keeps track of the services that have been added to a dynamic
+ * multi-scheduler. For each service entry, the developer is provided with
```

Comment:
Finish sentence?

Reply:
Whoops, will merge for now and fix in a following PR